stdlib/public/SDK/Foundation/NSCoder.swift

```diff
@@ -145,6 +145,13 @@ extension NSKeyedUnarchiver {
   }
 
   @nonobjc
+  @available(swift, obsoleted: 4)
+  @available(OSX 10.11, iOS 9.0, *)
+  public class func unarchiveTopLevelObjectWithData(_ data: Data) throws -> AnyObject? {
```

Comment:
Should this be named unarchiveTopLevelObject(with data: Data) throws -> AnyObject? to fit the naming guidelines?